package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "price-feed-poller",
```

Comment:
I find the name change completely unnecessary

Reply:
This comes from the proposal or renaming the witnet/witnet-price-feed-examples repo as to witnet/witnet-price-feeds, as its purpose is not as much as providing examples but gather instead all the "official" price feeds currently sustained by the Witnet Foundations. 

Being so, it makes sense to me that the "poller" agent intended to work with those price feeds be called after witnet/witnet-price-feeds-poller. A secondary effect is that these two repos would get alphabetically listed one after the other.